static/app/views/starfish/queries/useSpanSamples.tsx

```diff
@@ -52,6 +67,8 @@ export const useSpanSamples = (options: Options) => {
         `${url}?${qs.stringify({
           ...dateCondtions,
           ...{utc: location.query.utc},
+          upperBound: computeAxisMax([spanMetricsSeriesData?.[`p95(${SPAN_SELF_TIME})`]]),
+          lowerBound: 0,
```

Comment:
Fyi @wmak I’m curious, why is the lowerBound auto set to p50 in the backend? In this PR I set my lowerBound to 0, as i’m fairly sure the yAxis on the chart always starts at 0. But if there's a reason to do p50 lmk!

Reply:
nope 0 makes sense, p50 is the lowest percentile we can do in the metrics dataset

Reply:
fyi @wmak, updated this to use `lower`, `upper`, `firstBound` and `secondBound`